Assets/Scripts/UI/TooltipView.cs

```diff
@@ -1,21 +1,99 @@
-using System.Collections;
+using System;
 using System.Collections.Generic;
 using UnityEngine;
+using UnityEngine.UI;
 using TMPro;
 using Schemas.Actions;
 
 namespace UI
 {
     public class TooltipView : View
     {
+        /// <summary>
+        /// Holds all information that would be displayed in a tooltip.
+        /// </summary>
+        [Serializable]
+        public class TooltipDescription
```

Comment:
TooltipData? Description is overloaded in one of the members, aka `public string description`

Reply:
What if we just moved all this data like title, description, sprite, actions into the structure schema, and this TooltipDescription can instead be a struct that is NOT serializable, and we just new one up and pass the data through when creating the tooltip